spring-core/src/main/java/org/springframework/util/MimeType.java

```diff
@@ -583,7 +581,7 @@ public int compareTo(MimeType other) {
 				}
 			}
 		}
-
+		
```

Comment:
I assume this will break the build, since we do not allow trailing whitespace.

In any case, please remove those tabs.

Reply:
I just removed it